samples/nrf9160/fmfu_smp_svr/update_modem.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import argparse
+import logging
+import time
+import os
+from pynrfjprog import HighLevel
+
+def run(uart, modem_firmware_zip, baudrate):
+    if not os.path.exists(modem_firmware_zip):
+        raise Exception("Modem firmware zip file does not exist")
+    print('# nrf9160 modem firmware upgrade over serial port example started.')
+
+    # Configure logging to see HighLevel API output
+    logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+    # Find the hex file to flash. It should be located in the same directory as
+    # this example file.
+    with HighLevel.API() as api:
+        start_time = time.time()
+        with HighLevel.ModemUARTDFUProbe(api, uart, baudrate) as modem_dfu_probe:
+            modem_dfu_probe.program(modem_firmware_zip)
+            modem_dfu_probe.verify(modem_firmware_zip)
+
+        # Print upload time
+        print("------------------------------------------------------------")
+        print("The operation took {0:.2f} seconds ".format(time.time() - start_time))
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
```

Comment:
Could you try to make this file max 80 chars wide?

Reply:
Sure, but I don't see a great benefit in it :D

Reply:
I agree, its just that its easier to have them 80-chars in the first place ;)